ob2/web/blueprints/dashboard/__init__.py

```diff
@@ -29,6 +29,7 @@
 from ob2.util.security import require_csrf_token
 from ob2.util.time import now_compare, slip_units_now
 from ob2.util.validation import fail_validation, ValidationError, redirect_with_error
+from ob2.util.job_limiter import limit_queued_job, MAX_JOBS_ALLOWED
```

Comment:
We should make MAX_JOBS_ALLOWED configurable. (see config/config.yaml)

Also, shouldn't this default to the current behavior? (unlimited)